### DISCUSSION TITLE
Flying Star Zi Wei Dou Shu

### DISCUSSION THREAD

**Discussion by ameurenand** [SylarLong/iztro#181] [Ideas]
Hello,

I'm a SiHua FeiXing Zi Wei Dou Shu practitioner and I would like to know if it's possible to improve your library to handle the Flying Star Zi Wei Dou ?

**Comment by SylarLong** [comment]
@ameurenand 
Hello,

Nice to meet you. The flying star feature has been implemented in the lib. if the approach that you learned are different from iztro. Please try to use `plugin` feature to enhance your code. If you have any idea feel free to create a PR.

here's the document for plugin development: https://ziwei.pro/posts/config-n-plugin.html#%E6%8F%92%E4%BB%B6 . I'm sorry there's only Chinese version for now. if you're interested in it. please send an email to me. we can add each other on social media apps. my email address is : sylarlong@gmail.com
